app/assets/javascripts/views/story_view.js

```diff
@@ -710,5 +715,11 @@ module.exports = FormView.extend({
 
   clickSave: function(event) {
     this.saveEdit(event, false);
+  },
+
+  attachmentStart: function() {
+    var $storyControls = this.$el.find('.story-controls');
+    $storyControls.find('.submit, .destroy, .cancel').prop('disabled', true);
```

Comment:
Do we need this line here? As its present in the `render` method, maybe you can only call `render()` after setting `this.uploadInProgress` to true.